provision/testdata/help.stdout.txt

```diff
@@ -2,7 +2,7 @@ usage: acc_provision.py [-h] [-v] [--release] [--debug] [--sample] [-c file]
                         [-o file] [-z file] [-r file] [-a] [-d] [-u name]
                         [-p pass] [-w timeout] [--list-flavors] [-f flavor]
                         [-t token] [--test-data-out file] [--skip-kafka-certs]
-                        [--upgrade] [--disable-multus disable_multus]
+                        [--upgrade] [--disable-multus disable_multus] [--helm]
```

Comment:
Could we have two flags here:
1. [--helm-values file]
2. [--helm-chart file] = a generated aci-helm.tar.gz chart (equivalent to output of `helm package`)

Or 
1. [--helm dir] which generates both files at the dir